### PR TITLE
RIA-3720

### PR DESCRIPTION
**Pull request by Lee365**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-3720

### Change description ###
RIA-3720 - AO lists the case (remote hearing)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -67,6 +67,11 @@ hearingNoticeDocument.fileExtension: PDF
 hearingNoticeDocument.fileName: "hearing-notice"
 hearingNoticeDocument.templateName: ${IA_HEARING_NOTICE_TEMPLATE:TB-IAC-HNO-ENG-00241.docx}
 
+remoteHearingNoticeDocument.contentType: application/pdf
+remoteHearingNoticeDocument.fileExtension: PDF
+remoteHearingNoticeDocument.fileName: "hearing-notice"
+remoteHearingNoticeDocument.templateName: ${IA_REMOTE_HEARING_NOTICE_TEMPLATE:Remote_hearing_notice.docx}
```
Comment (bradsorour) [inline src/main/resources/application.yaml:73]: This template name will need to change as soon as uploaded to SharePoint.

Comment (Lee365) [reply src/main/resources/application.yaml]: Template name updated.

**Review by bradsorour — DISMISSED** [review]
LGTM

**Review by bradsorour — APPROVED** [review]
LGTM

**Review by makowalski — APPROVED** [review]
LGTM

**Pull request merged**